reviewer-guide/docs/01-Review/01-identify-changes-in-framework.md

```diff
@@ -0,0 +1,140 @@
+
+The version number follows the Semantic Versioning Specification (SemVer).
+
+## Semantic Versioning (SemVer)
+
+SemVer attempts to provide more meaning of the underlying changes in a project through additional structure to version numbers. 
+
+Each Version X.Y.Z must be non-negative numbers starting and (supposed to) have non-leading zeroes (though many libraries start at `0.1.0`).
+
+- X is a **MAJOR** version
+
+- Y is a **MINOR** version
+
+- Z is a **PATCH** version
+
+With these versions specified, `0.2.10` is ordered before `0.3.1` and subsequently, `1.0.0` comes after both `0.2.10` and `0.3.1`.
+
+A PATCH version is incremented when internal bugs are fixed in a library that does not affect any projects. MINOR versions are incremented when there is 'backward compatible' features introduced to a library or significant improvements to the internal workings of the library are added. MAJOR versions are incremented when there are **Breaking Changes** introduced to the library.
+
+In the previously mentioned example, theoretically, there are backward compatible features added in-between the packages `0.2.10` and `0.3.1` versions. When the package reached `1.0.0`, it can be assumed that it contains breaking changes if you wrote your application on a previous version (e.g., `0.2.10` or `0.3.1`).
+
+### SemVer Version Ranges
+
+A lot of the time, specifying a single version of a technology is too fine grain and restrictive because subsequent MINOR and PATCH updates won't break the project that the dependency is in. It can be annoying to have to update version numbers _every_ time a MINOR or PATCH version is released.
+
+This is why SemVer has version ranges which can be denoted a few different ways. The simplest ways are to use a set of primitive operators, which are: `<` Less than, `<=` Less than or equal to, `>` Greater than, `>=` Greater than or equal to, and `=` Equal. `=` can be assumed when no operator is present. 
+
+For review, a range specifying `>15.3.0 <16.4.1` says the course started at version `15.3.0` and is valid up to all version (excluding) `16.4.1` where a `<=16.4.1` will specify the version including `16.4.1`.
+
+To make versions easier to specify, the squiggly `~` and caret `^` also identify ranges. When `~` is placed before version (e.g., `~15.0.0`), it specifies all versions incremented at the PATCH level. In other words `15.0.1`, `15.0.3`, or `15.0.19` are all valid. In other words, `~15.0.0` equals `>=15.0.0 <15.1.0`.
+
+The caret `^` specifies all version increments at the MINOR and PATCH levels. `^15.0.0` specifies that any of these following versions are valid: `15.0.1`, `15.1.3`, or `15.5.0`. in other words, `^15.0.0` would equal the range `>=15.0.0 <16.0.0`.
+
+### Fine-grain Version Specification
+
+The `||` operator combines versions and version ranges. `^15 || ^16` specifies all MINOR and PATCH version increments in the MAJOR versions `15` and `16`.
```

Comment:
This might be the cleanest way to specify a range in the way we want.

Reply:
`15 - 16` the `^` is tautological and it is a contiguous range